frontend/app/model/MembersOnlyContent.scala

```diff
@@ -0,0 +1,31 @@
+package model
+
+import com.gu.contentapi.client.model.{Asset, Content}
+
+case class MembersOnlyContent(content: Content) {
```

Comment:
Yeah thats right but it felt as if a lot was going on in the controller so moved it from service to model which felt a little better. But I can put it in the controller if wanted?